strongbox-web-core/src/main/java/org/carlspring/strongbox/controller/ArtifactController.java

```diff
@@ -73,11 +73,12 @@ public ResponseEntity greet()
     @ApiResponses(value = { @ApiResponse(code = 200, message = "The artifact was deployed successfully."),
                             @ApiResponse(code = 400, message = "An error occurred.") })
     @PreAuthorize("hasAuthority('ARTIFACTS_DEPLOY')")
-    @RequestMapping(value = "{storageId}/{repositoryId}/**", method = RequestMethod.PUT)
+    @RequestMapping(value = "{storageId}/{repositoryId}/{subPath:.+}", method = RequestMethod.PUT)
```

Comment:
Okay,  it's great that you can specify any path variable name! :)
It's just that everywhete else it's called `path`... Even across the service and util classes and this way it will be more consistent across the codebas.